src/Analyzers/VisualBasic/Tests/NamingStyles/NamingStylesTests.vb

```diff
@@ -19,10 +19,6 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics.Naming
             Return (New VisualBasicNamingStyleDiagnosticAnalyzer(), New NamingStyleCodeFixProvider())
         End Function
 
-        Protected Overrides Function GetComposition() As TestComposition
```

Comment:
[this is a comment to the file itself.]

Did we want some of the tests that exist on the C# side of things here as well?